spec/features/tasks/edit_spec.rb

```diff
@@ -10,17 +10,14 @@
     click_on 'Edit'
     expect(page).to have_content('Editing task')
 
-    fill_in 'Name', with: 'Test my app (updated)'
-    click_button 'Save'
-
+    fill_name_and_save('Test my app (updated)')
```

Comment:
[suggestion] Could you use a factory on line 6 to create a task for this spec?

Reply:
Good suggestion. Modified.